tests/CMakeLists.txt

```diff
@@ -196,3 +196,65 @@ add_test(
 )
 set_tests_properties("test-generate_contributors" PROPERTIES LABELS quick)
 endif()
+
+if (PYTHON3_EXECUTABLE AND SSG_PRODUCT_RHEL8)
+    add_test(
+        NAME "refchecker-rhel8-cis_workstation_l1"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/utils/refchecker.py" --json "${CMAKE_SOURCE_DIR}/build/rule_dirs.json" --build-config-yaml "${CMAKE_BINARY_DIR}/build_config.yml" "rhel8" "cis_workstation_l1" "cis"
+    )
+    set_tests_properties("refchecker-rhel8-cis_workstation_l1" PROPERTIES FIXTURES_REQUIRED "rule-dir-json")
+    set_tests_properties("refchecker-rhel8-cis_workstation_l1" PROPERTIES DEPENDS "test-rule-dir-json")
+    set_tests_properties("refchecker-rhel8-cis_workstation_l1" PROPERTIES LABELS quick)
```

Comment:
Can we create a macro for the tests that call refchecker.py? That would reduce the repeating code.